test.js

```diff
@@ -17,12 +18,28 @@ describe('Test Fixtures', () => {
 
   for (let fixture of fixtures) {
     test(fixture, async () => {
+      let writeFileSync;
+      if (fixturesWithFix.includes(fixture)) {
+        process.argv.push('--fix');
+        writeFileSync = fs.writeFileSync;
+        fs.writeFileSync = (filePath, updatedTranslations, encoding) => {
+          let fileName = filePath.split('/').slice(-2).join('/');
+          log();
+          log(fileName);
+          log(updatedTranslations);
+          log(encoding);
```

Comment:
instead of using `log()` here, I would have something like `let writtenFiles = new Map()`, then `writtenFiles.set(fileName, content)` and finally `expect(writtenFiles).toMatchSnapshot()`. does that make sense?

Reply:
I have added this, but not sure if i should add a conditional for the `expect(writtenFiles).toMatchSnapshot()` currently i have two snapshots for each, i was considering adding `expect.assertions` but i would need to also put the same conditional around that to set it to 2 or 3 so not sure about it